sky/task.py

```diff
@@ -948,7 +948,7 @@ def _get_preferred_store(
         store_type = storage_lib.StoreType.from_cloud(storage_cloud_str)
         return store_type, storage_region
 
-    def sync_storage_mounts(self) -> None:
+    def sync_storage_mounts(self, force_sync: bool = False) -> None:
```

Comment:
Can we add a quick docstr for `force_sync`, mentioning under what conditions storages are not synced and when force_sync may be required?